src/onnx/parse_dynamicquantizelinear.cpp

```diff
@@ -98,41 +98,40 @@ struct parse_dynamicquantizelinear : op_parser<parse_dynamicquantizelinear>
         if(x_shape.dynamic())
             MIGRAPHX_THROW("DYNAMICQUANTIZELINEAR: dynamic shapes are not supported");
 
-        auto x_reshaped =
-            (x_shape.lens().size() == 1)
-                ? x
-                : info.add_instruction(
-                      migraphx::make_op("reshape", {{"dims", {x_shape.elements()}}}), x);
-
         auto lit_0 = info.add_literal(migraphx::literal{migraphx::shape{x_type}, {0}});
-        x_reshaped =
-            info.add_instruction(migraphx::make_op("concat", {{"axis", 0}}), x_reshaped, lit_0);
-
         // 1. Computing y_scale
         // Note: currently, DynamicQuantizeLinear only has uint8 quantization:
-        const auto x_max = std::numeric_limits<uint8_t>::max();
-        const auto x_min = std::numeric_limits<uint8_t>::min();
-
-        auto q_range =
-            info.add_literal(migraphx::literal{migraphx::shape{x_type}, {x_max - x_min}});
+        const auto type_max = std::numeric_limits<uint8_t>::max();
+        const auto type_min = std::numeric_limits<uint8_t>::min();
+        std::vector<size_t> len_vec(x_shape.lens().size());
```

Comment:
Can you name this `axes` since its used for the axes to the reducitons?

Reply:
Done